src/service/resources-impl.js

```diff
@@ -1202,7 +1202,7 @@ export class ResourcesImpl {
     this.relayoutAll_ = false;
     const relayoutTop = this.relayoutTop_;
     this.relayoutTop_ = -1;
-    const elementsThatScrolled = this.elementsThatScrolled_.splice(0, Infinity);
+    const elementsThatScrolled = this.elementsThatScrolled_;
```

Comment:
Minor: Replace this local var with `this.elementsThatScrolled_` for one less indirection.